target/linux/ipq806x/base-files/etc/board.d/01_leds

```diff
@@ -45,6 +45,15 @@ nbg6817)
 	ucidef_set_led_wlan "wlan5g" "WLAN5G" "$board:amber:wifi5" "phy0radio"
 	ucidef_set_led_netdev "wan" "WAN" "$board:white:internet" "eth1"
 	;;
+vr2600v)
+	ucidef_set_led_default "dsl" "DSL" "${board}:white:dsl" "0"
```

Comment:
Drop all instances of ucidef_set_led_default where you switch off LEDs and tell me which LEDs are on by default albeit they should be off.